src/pages/publication/PublicationPage.tsx

```diff
@@ -30,7 +67,72 @@ const PublicationPage: FC<PublicationPageProps> = ({ publicationId }) => {
   }, [dispatch, publicationId]);
 
   return (
-    <>{isLoadingPublication ? <CircularProgress color="inherit" size={20} /> : <p>Retrieved: {publication.id}</p>}</>
+    <>
+      {isLoadingPublication ? (
+        <CircularProgress color="inherit" size={20} />
+      ) : (
+        <ContentPage>
+          {publication?.id ? (
```

Comment:
syns det er litt rart å ha en sjekk på id her, hva med å bruke `null` som default state og bare ha en sjekk for om vi har publikasjon her?